map/bookmark_helpers.cpp

```diff
@@ -48,6 +48,14 @@ std::map<std::string, BookmarkMatchInfo> const kFeatureTypeToBookmarkMatchInfo =
   {"amenity-pub", {kml::BookmarkIcon::Pub, BookmarkBaseType::Food}},
   {"amenity-cafe", {kml::BookmarkIcon::Cafe, BookmarkBaseType::Food}},
 
+  {"amenity-bbq", {kml::BookmarkIcon::Food, BookmarkBaseType::Food}},
+  {"amenity-restaurant", {kml::BookmarkIcon::Food, BookmarkBaseType::Food}},
+  {"leisure-picnic_table", {kml::BookmarkIcon::Food, BookmarkBaseType::Food}},
+  {"tourism-picnic_site", {kml::BookmarkIcon::Food, BookmarkBaseType::Food}},
+
+  {"amenity-fast_food", {kml::BookmarkIcon::FastFood, BookmarkBaseType::Food}},
+  {"amenity-food_court", {kml::BookmarkIcon::FastFood, BookmarkBaseType::Food}},
```

Comment:
Not sure if we should treat a food_court as a fast food in general ..
IMO, each vendor _should/used to_ be mapped separately in a food court, no?

Reply:
_Fast_ food in food courts usually means that food is prepared quickly, without longer waiting times. So, from this perspective, it looks okay.

Vendors ideally should be mapped separately too.

Reply:
Well, I suspect that food court doesn't mean all vendors are fast food. So "fork and knife" icon for a generic food court is better, IMO.

Reply:
In this commit I used the fast_food icon for food_court bookmark icons because the food_court POIs currently use the fast_food icon. 

Since you seem to agree that the fork-and-knife icon is better for food_courts, I've made a new commit changing the food_court POI icon to the restaurant icon.